crates/bevy_color/src/lib.rs

```diff
@@ -0,0 +1,89 @@
+//! Representations of colors in various color spaces.
+//!
+//! This crate provides a number of color representations, including:
+//!
+//! - [`Srgba`] (standard RGBA, with gamma correction)
+//! - [`LinearRgba`] (linear RGBA, without gamma correction)
+//! - [`Hsla`] (hue, saturation, lightness, alpha)
+//! - [`Lcha`] (lightness, chroma, hue, alpha)
+//! - [`Oklaba`] (lightness, a-axis, b-axis, alpha)
+//!
+//! Each of these color spaces is represented as a distinct Rust type.
+//! Each of these color spaces is represented as a distinct Rust type.
+//!
+//! # Color Space Usage
+//!
+//! Rendering engines typically use linear RGBA colors, which allow for physically accurate
+//! lighting calculations. However, linear RGBA colors are not perceptually uniform, because
+//! both human eyes and computer monitors have non-linear responses to light. "Standard" RGBA
+//! represents an industry-wide compromise designed to encode colors in a way that looks good to
+//! humans in as few bits as possible, but it is not suitable for lighting calculations.
+//!
+//! Most image file formats and scene graph formats use standard RGBA, because graphic design
+//! tools are intended to be used by humans. However, GPU hardware operates on linear RGBA, so it
```

Comment:
I don't think GPU hardware _specifically_ cares about linear rgb vs srgb. It will happily operate on either representation. Our rendering code requires linear rgb because that is what it expects. We _could_ write an srgb renderer, it would just suffer from that choice. The expected output format for the window surface / swapchain is platform specific and often it can be configured to use either srgb or linear formats.

Reply:
Comment updated.